File: 13/py/main.py
def countZeroes(array):
    left = 0
    right = len(array) - 1
    middle = 0

    while left <= right:
        middle = (left + right) // 2

        if array[middle] == 0:
            right = middle - 1
        else:
            left = middle + 1

    if array[middle] == 1:
        middle += 1

    return len(array) - middle

File: 13/go/main.go
package main

func countZeroes(array []int) int {
	var (
		left   = 0
		right  = len(array) - 1
		middle = 0
	)

	for left <= right {
		middle = (left + right) / 2
		if array[middle] == 1 {
			left = middle + 1
		} else {
			right = middle - 1
		}
	}

	if array[middle] == 1 {
		middle++
	}

	return len(array) - middle
}
